src/components/networked.js

```diff
@@ -267,6 +287,18 @@ AFRAME.registerComponent('networked', {
     if (entityData[0] == 1) {
       entityData = Compressor.decompressSyncData(entityData, this.getAllSyncedComponents());
     }
+
+    if (entityData.lastOwnerTime < this.lastOwnerTime) {
```

Comment:
You are checking the opposite condition bellow`this.lastOwnerTime < entityData.lastOwnerTime` so I don't think this check is needed.